<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenRGBDialog2Ui</class>
 <widget class="QMainWindow" name="OpenRGBDialog2Ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenRGB</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0" colspan="5">
     <widget class="QTabWidget" name="MainTabBar">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <widget class="QWidget" name="TabDevices">
       <attribute name="title">
        <string>Devices</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QTabWidget" name="DevicesTabBar">
          <property name="tabPosition">
           <enum>QTabWidget::West</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="TabInformation">
       <attribute name="title">
        <string>Information</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTabWidget" name="InformationTabBar">
          <property name="tabPosition">
           <enum>QTabWidget::West</enum>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="7" column="0" rowspan="2" colspan="5">
     <layout class="QHBoxLayout" name="MainButtonsLayout">
      <item>
       <widget class="QPushButton" name="ButtonRescanDevices">
        <property name="text">
         <string>Rescan Devices</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonSaveProfile">
        <property name="text">
         <string>Save Profile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonLoadProfile">
        <property name="text">
         <string>Load Profile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonDeleteProfile">
        <property name="text">
         <string>Delete Profile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ProfileBox"/>
      </item>
     </layout>
    </item>
    <item row="4" column="0" rowspan="3" colspan="5">
     <layout class="QHBoxLayout" name="DetectorLayout">
      <item>
       <widget class="QLabel" name="DetectionProgressLabel">
        <property name="text">
         <string>OpenRGB is detecting devices...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="DetectionProgressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonStopDetection">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
